docs/source/clients/cql.md

```diff
@@ -0,0 +1,104 @@
+# Using CQL
+
+## Embedded cqlsh (aka localhost)
+
+Every ScyllaDB node has an integrated `cqlsh` available. Here is an example of how it can be used.
+
+::::{tab-set}
+:::{tab-item} Any ScyllaDB Node
+```bash
+kubectl exec -it service/<sc-name>-client -c scylla -- cqlsh 
+```
+:::
+:::{tab-item} Specific ScyllaDB Node
+```bash
+kubectl exec -it pod/<sc-name>-<datacenter>-<node-index> -c scylla -- cqlsh
+```
+:::
+::::
+```text
+Connected to scylla at 127.0.0.1:9042
+[cqlsh 6.2.0 | Scylla 5.4.0-0.20231205.58a89e7a4231 | CQL spec 3.3.1 | Native protocol v4]
+Use HELP for help.
+cqlsh>
+```
+
+## Remote cqlsh
+
+This paragraph shows how to use `cqlsh` to remotely connect to a ScyllaDB node.
+It is strongly recommended to access CQL over TLS connections on port `9142` instead of unencrypted `9042`.
+Note that Scylla Operator sets up TLS certificates by default and makes them accessible in the Kubernetes API,
+so the encrypted port `9142` works by default.
+(In future releases the unencrypted port `9042` will be disabled by default, unless explicitly opted-in.)  
```

Comment:
I'd skip the parentheses. If you want it to stand out, maybe use a attention/caution or whatever other admonition?